api/python/gdh.py

```diff
@@ -0,0 +1,60 @@
+# Depends: requests pandas
+# You may already have these installed, otherwise install using pip:
+#     pip install requests pandas
+
+# Example invocations:
+# >>> from gdh import cases
+# >>> c1 = cases("country:'New Zealand'", "API KEY HERE")
+# >>> c2 = cases("country:'New Zealand' admin1:Auckland", "API KEY HERE")
+#
+# You can use the cases_cached() function to cache the cases locally.
+# This is useful for rerunning the script which will then use the
+# cached version.
+
+import os
+import io
+import json
+import hashlib
+
+import requests
+import pandas as pd
+
+GH = "https://data.covid-19.global.health"
+downloadAsync = "/api/cases/downloadAsync"
+
+def cases(query, apikey, server=GH):
+    res = requests.post(
+        f"{server}{downloadAsync}",
+        data=json.dumps({"format": "csv", "query": query}),
+        headers={"Content-Type": "application/json", "X-API-Key": apikey},
+    )
+    if res.status_code != 200:
+        print(res.text)
+        return False
```

Comment:
I think a `raise` might be better here, so the return value of the function is of a single type